source/renderer/app/components/wallet/wallet-import/WalletImportFileDialog.js

```diff
@@ -121,10 +121,27 @@ export default class WalletImportFileDialog extends Component<Props, State> {
   importPathInput: Input;
 
   componentWillMount() {
-    // Reset migration data
     this.props.onOpen();
   }
 
+  handleSelfRef = (ref: ?ElementRef<'div'>) => {
+    if (ref) {
+      this.applyDialogStyles();
+    }
+  };
+
+  applyDialogStyles = () => {
+    const dialogElement = window.document.querySelector('.ReactModal__Content');
+    const dialogOverlayElement = dialogElement.parentElement;
+    dialogOverlayElement.style.backgroundColor =
+      'var(--theme-wallet-import-background-color)';
+    dialogElement.style.backgroundColor = 'transparent';
+    dialogElement.style.border = 'none';
+    dialogElement.style.boxShadow = 'none';
+    dialogElement.style.height = '100%';
+    dialogElement.style.width = '100%';
+  };
```

Comment:
@DeeJayElly do we really need this. Can we apply style like `:global` for specific case without writing styles inside JSX code?
If we can isolate that then we also don't need refs anymore because they are not recommended in react

Reply:
Well i tried to override it without this approach with JSX, but unfortunately i couldn't find a way to do it since this ReactModal is basically React Polymorph component.

Reply:
@DeeJayElly then introduce new property `themeOverrides` in `widgets/dialog` and pass overrides to RP Modal component. + define specific scss overrides for Dialog defined in Import / Migrate components and that is it

Reply:
@DeeJayElly here is an example how to do that 
https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/components/wallet/settings/WalletRecoveryPhraseVerificationWidget.js#L262

Reply:
Implemented ;)